story-generator/src/app/proving-grounds/proving-grounds.css

```
@@ -0,0 +1,896 @@
+
+.form-control:focus {
+  outline: none;
+  border-color: #a855f7;
+  box-shadow: 0 0 10px rgba(168, 85, 247, 0.3);
+}
+
+.form-control-range {
+  width: 100%;
+  padding: 0;
+  appearance: none;
+  background: transparent;
```

Stylelint (17.12.0)

[error] Duplicate property "background" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)